app/components/settings/billing-page/payment-history-section.hbs

```diff
@@ -0,0 +1,66 @@
+<Settings::FormSection @title="Payment history" @description="Details of your previous payments">
+  {{#if this.isLoading}}
+    <div class="flex justify-center py-8">
+      <div class="animate-spin rounded-full h-8 w-8 border-b-2 border-gray-900"></div>
+    </div>
+  {{else if this.errorMessage}}
+    <div class="bg-red-50 border border-red-200 rounded-md p-4">
+      <div class="flex">
+        <div class="flex-shrink-0">
+          <svg class="h-5 w-5 text-red-400" viewBox="0 0 20 20" fill="currentColor">
```

Comment:
@Arpan-206 what is this SVG? Check other places to see how we usually use SVGs - we use `svg-jar` to pull them in. Prefer to re-use something we have rather than adding new ones